admin-dev/themes/default/template/footer.tpl

```diff
@@ -52,7 +52,7 @@
 	</div>
 	<div class="col-sm-5">
 		<div class="footer-contact">
-			<a href="http://www.prestashop.com/en/contact_us?utm_source=back-office&amp;utm_medium=footer&amp;utm_campaign=back-office-{$lang_iso|upper}&amp;utm_content={if $host_mode}cloud{else}download{/if}" class="footer_link _blank">
+			<a href="https://www.prestashop.com/en/contact_us?utm_source=back-office&amp;utm_medium=footer&amp;utm_campaign=back-office-{$lang_iso|upper}&amp;utm_content={if $host_mode}cloud{else}download{/if}" class="footer_link _blank">
```

Comment:
Should I remove the `/en` here as well?

Reply:
Yes, and it works if I change `contact_us` to `contact`

Reply:
```suggestion
			<a href="https://www.prestashop.com/contact_us?utm_source=back-office&amp;utm_medium=footer&amp;utm_campaign=back-office-{$lang_iso|upper}&amp;utm_content={if $host_mode}cloud{else}download{/if}" class="footer_link _blank">
```